### ISSUE TITLE
[Suggestion] Swiss Army Hammer

### ISSUE THREAD

**Issue by harald921** [CleverRaven/Cataclysm-DDA#14725]
http://www.watchesgood.com/products/21063.jpg

Adding a tool that servers as an upgrade from the swiss army knife, yet not as good as the toolbox. Rare of course, perhaps even a crafting recipe. What are your thoughts on this?

**Comment by kevingranade** [comment]
That's *really* niche and silly.
That kind of tool works terribly in practice.

**Comment by Mecares** [comment]
Have to agree, i would absolutely hate to hammer a few dozen thick nails into quite hard wood with a thing like this but it is better than with bare hands.

**Comment by ValidAQ** [comment]
Yeah, these things are pretty ineffective compared to ordinary hammers and multi-tools.
Wouldn't see much use.
Besides, by the point you find/can make one, you're likely to already have all those tools, anyway.

**Comment by Malkeus** [comment]
It looks liek it would be dangerous to the user tbh. Lots of pinch points on there.

**Comment by harald921** [comment]
I actually have one of these IRL. They aren't half as impractical as they seem. For hammering that is. However, it's not meant to be some 'super tool', just something that you could use for disassembling stuff + some bonus features.

**Comment by ValidAQ** [comment]
I don't know, I have one too, as well as ordinary hammer and mutli-tool. When hammering is required, it just feels noticeably more awkward than a hammer, and with screw driving/cutting/sawing, multi-tool performs better.
Not saying I'm against adding it - just, it's not unusable, but fairly situational.

**Comment by Mecares** [comment]
Well comments about how horrible it is to use one, not necessarily mean that it should not be added, but it should be less good than the separate dedicated tools at what they do.

**Comment by harald921** [comment]
@ValidAQ I'd myself take a normal hammer over this any day of the week, however if I am going camping I would grab the multi-tool hammer. Hammering is okay-ish, the "bolt turning" is kinda awkward, and the other tools are kinda horrible - but it's better than nothing!

Also, I agree with @Mecares

**Comment by chaosvolt** [comment]
I get the feeling that whether they're a piece of junk depends highly on brand. Probably would be especially leery of the versions that appear in stores as part of the "christmas gift gadgets" stuff. :V

Personally IRL I prefer to just stick with a basic pair of pliers in my backpack, pen screwdriver set, and a couple other things. Haven't yet needed to always carry a hammer on my person.

The idea for this as in-game thing, though? Depends on what it offers relative to the multitool and swiss army knife. While you can easily make a stone/makeshift hammer, the same goes for knives as well, so the other two tools are already non-craftable tools that combine an easily-made essential crafting quality with a few less easily-obtained qualities with more specific uses.

So basically a convenience item depending on what tools you've found, and what you regularly need.

**Comment by kevingranade** [comment]
The difficulty in balancing this kind of thing is that every time a "worse"
version of a tool is introduced it means shuffling numbers around in a lot
of recipes.
Similarly it might fit between "rock" and "hammer", and if there isn't an
existing level there, adjustments will need to be made.
If it's at rock level though, that's just fine.

**Comment by chaosvolt** [comment]
> If it's at rock level though, that's just fine.

If it's at rock level, then it kinda has very little reason for existence. Once you have a rock, you have the only survival tool you'll ever need. o3o

**Comment by mugling** [comment]
closing as appears to be stalled
